dev-python/twisted/twisted-21.2.0.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=7
 
 DISTUTILS_USE_SETUPTOOLS=rdepend
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{7..10} )
```

Comment:
Does twisted pass for you?

I get:
```
  NewFetchTests
    testFetchAll ...                                                    [ERROR]
    testFetchAllUID ...                                                 [ERROR]
                                                [ERROR]
    testFetchBody ...                                                   [ERROR]
                                                  [ERROR]
    testFetchBodyParts ...                                              [ERROR]
                                             [ERROR]
    testFetchBodyStructureUID ...                                       [ERROR]
                                      [ERROR]
    testFetchBodyUID ...                                                [ERROR]
                                               [ERROR]
    testFetchEnvelope ... 
```
and at this point it hangs.

Reply:
Strange. Yes, these do pass for me.

Reply:
Yes.